test/clj_sqs_extended/core_test.clj

```diff
@@ -27,14 +29,14 @@
   (testing "Sending a standard message with a nil body yields exception"
     (fixtures/with-test-standard-queue
       (is (thrown? Exception
-                   (sqs-ext/send-message @fixtures/test-sqs-ext-client
+                   (sqs-ext/send-message fixtures/aws-config
```

Comment:
this looks cleaner

Reply:
We still have that fixture improvement issue open, so there is still potential for cleaning this up even more.